user/languages/smalltalk.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Building a Smalltalk Project
+layout: en
+permalink: /user/languages/smalltalk/
+---
+
+### What This Guide Covers
+
+This guide covers build environment and configuration topics specific to Smalltalk
+projects. Please make sure to read our
+[Getting Started](/user/getting-started/) and
+[general build configuration](/user/customizing-the-build/) guides first.
+We currently just support [Squeak/Smalltalk](http://squeak.org/).
+
+### Community-Supported Warning
+
+Travis CI support for Smalltalk is contributed by the community and may be removed or
+altered at any time. If you run into any problems, please report them in the
+[Travis CI issue tracker](https://github.com/travis-ci/travis-ci/issues/new?labels=community:crystal).
```

Comment:
The URL should be https://github.com/travis-ci/travis-ci/issues/new?labels=community:smalltalk and should mention to cc responsible users.